### PR TITLE
Refactor code to meet naming conventions

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
Also, removed usage of `snyk monitor` as part of CI. Github/snyk integration now handles this.

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-174.dev.beta.nhschoices.net)

File: .travis.yml [hunk 1]
```diff
@@ -10,12 +10,15 @@ before_install:
 install:
   - sudo sysctl -w vm.max_map_count=262144
 
-before_script: greenkeeper-lockfile-update
+before_script:
```
Comment (st3v3nhunt) [inline .travis.yml]: Why the change here (and the other 2 instances)?

I've been moving to having single lines as they are in order to reduce the length of the file.

File: docker-compose-tests.yml [hunk 2]
```diff
@@ -15,17 +15,16 @@ services:
     links:
       - gp-finder-db:es
 
-  travis-tests:
+  ci-tests:
```
Comment (st3v3nhunt) [inline docker-compose-tests.yml]: Can this be renamed to `test-ci` - that is what I've done across the rest of the repos.

File: docker-compose-tests.yml [hunk 3]
```diff
@@ -15,17 +15,16 @@ services:
...
     extends:
       file: docker-compose.yml
       service: base-app
-    command: sh ./scripts/container-travis-tests
+    command: scripts/container/ci-tests
     volumes:
-      - gp-finder-travis-tests-node_modules:/code/node_modules
+      - gp-finder-ci-tests-node_modules:/code/node_modules
```
Comment (st3v3nhunt) [inline docker-compose-tests.yml]: If the service is renamed to `test-ci` I'd suggest renaming this to match i.e. `gp-finder-test-ci-node_modules` (not that it really matters...).

File: package.json [hunk 4]
```diff
@@ -5,26 +5,24 @@
...
-    "prepush": "yarn git-hook && yarn snyk -- test",
+    "lint-watch": "esw --watch .",
     "postrewrite": "yarn git-hook",
-    "snyk-travis": "yarn snyk -- monitor --org=nhsuk",
+    "precommit": "yarn git-hook",
+    "prepush": "yarn git-hook && yarn snyk test",
     "start": "node app.js",
+    "start-watch": "nodemon app.js | ./node_modules/bunyan/bin/bunyan & nodemon -e scss -x yarn build-css -- compact",
     "test": "NODE_ENV=test mocha --recursive test",
+    "test-ci": "yarn lint && yarn coverage-generate",
```
Comment (st3v3nhunt) [inline package.json]: Does this command need `&& yarn coverage-check && coverage-upload` adding to it?

File: scripts/README.md [hunk 5]
```diff
@@ -4,9 +4,8 @@ The scripts in these folder are meant to help the user/contributor to get up and
...
 
-[`ci-deployment`](ci-deployment) - these scripts are used behind the scenes to continually deploy the application
-to our development environment
+[`start`](start) - will start the application within the container and restart on file changes: `http://localhost:3000`
 
-[`start`](start) - will start the application in dev mode: `http://localhost:3000`
+[`test`](test) - will run the tests continually within the container.
 
-[`test`](test) - will continually run the tests
+[`test-ci`](test-ci) - will runs the tests within the container in CI. Will lint the code, run all the tests and generate code coverage stats
```
Comment (st3v3nhunt) [inline scripts/README.md]: Full stop missing?

File: scripts/container/README.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Container Scripts
+
+The scripts in these folder are meant to be called from docker compose files and are run within containers.
+
+[`ci-tests`](ci-tests) This script is run by docker compose within a container hosted in CI. Runs linter, tests and code coverage stats generation within the container.
+
+[`test-watch`](test-watch) Script to be run by docker compose within a container hosted locally. Runs tests within the container and watches for file changes.
+
```
Comment (st3v3nhunt) [inline scripts/container/README.md]: Unnecessary blank line?

File: scripts/test-ci [hunk 7]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# -p flag stands for --project-name and will create a different network for the test container
+# so it won't interfere with the app container
+docker-compose -p gp-finder-ci-test -f docker-compose-tests.yml down -v
+docker-compose pull gp-finder-db
+docker-compose -p gp-finder-ci-test -f docker-compose-tests.yml up -d gp-finder-db
+docker-compose -p gp-finder-ci-test -f docker-compose-tests.yml build ci-tests
```
Comment (st3v3nhunt) [inline scripts/test-ci]: If the comment about renaming `ci-tests` to `test-ci` is heeded this will need changing. And, just for consistency I'd suggest updating the project name flag to `gp-finder-test-ci` or maybe just `test-ci` - for the sake of consistency.

**Comment by neilbmclaughlin** [comment]
Since merging in from master the `git clone` method for getting the deployment scripts notifications are not appearing.

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-finder succeeded (http://gp-finder-pr-174.dev.beta.nhschoices.net)

**Comment by adamgraves-choices** [comment]
@neilbmclaughlin The stack in Rancher was in an "upgrading" state. Not sure why the deployment didn't remove the stack or fail...
